src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &--group > :nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
Copy all styles from previous task, cause what if I ask you to quickly mark 5 stars ?